sal_interfaces/ESS/ESS_Telemetry.xml

```diff
@@ -345,4 +345,50 @@
       <Count>1</Count>
     </item>
   </SALTelemetry>
+  <SALTelemetry>
+    <EFDB_Topic>ESS_accelerometer</EFDB_Topic>
+    <Description>LabJack connected accelerometer unit.</Description>
+    <item>
+      <EFDB_Name>sensorName</EFDB_Name>
+      <Description>The name of the sensor.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>timestamp</EFDB_Name>
+      <Description>Time at which the data was determined (TAI unix seconds).</Description>
+      <IDL_Type>double</IDL_Type>
+      <Units>second</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationX</EFDB_Name>
+      <Description>Acceleration in x direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationY</EFDB_Name>
+      <Description>Acceleration in y direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>accelerationZ</EFDB_Name>
+      <Description>Acceleration in Z direction.</Description>
+      <IDL_Type>float</IDL_Type>
+      <Units>m/s2</Units>
+      <Count>200</Count>
+    </item>
+    <item>
+      <EFDB_Name>location</EFDB_Name>
+      <Description>The location of the sensor.</Description>
```

Comment:
Consider:
Physical location of the sensor.

Reply:
This is from generic ESS description. If changed, it should be changed in all location fields. Let's do that in another PR

Reply:
At the end changed.

Reply:
Then why not change the timestamp one as well?